openmdao/docs/advanced_guide/derivs/partial_derivs_explicit.rst

```diff
@@ -0,0 +1,83 @@
+.. _advanced_guide_partial_derivs_explicit:
+
+OpenMDAO considers component derivatives to be **partial-derivatives**.
+The framework uses these partial-derivatives in order to compute the **total-derivatives** across your whole model.
+This tutorial is focused on how to define the partial-derivatives* for instances of :ref:`ExplicitComponent <comp-type-2-explicitcomp>`.
```

Comment:
extraneous asterisk:  partial-derivatives*  (was this meant to be bolded)

Reply:
also I don't believe "partial derivatives" and  "total derivatives" should be hyphenated